manager/src/main/java/com/redhat/service/smartevents/manager/BridgesService.java

```diff
@@ -11,7 +11,7 @@
 
 public interface BridgesService {
 
-    Bridge createBridge(String customerId, BridgeRequest bridgeRequest);
+    Bridge createBridge(String customerId, BridgeRequest bridgeRequest, String organisationId);
```

Comment:
@vaibhavjainwiz I'd personally put the `organisationId` parameter after the `customerId` parameter so the method logically reads:

"customer with id 'foo', from organisation 'bar' wants to create this Bridge"

Reply:
done